cmd/main.go

```diff
@@ -222,6 +220,43 @@ func initMC() {
 
 }
 
+func installAutoCompletion(ctx *cli.Context) {
+	if ctx.Bool("no-autocompletion") || ctx.GlobalBool("no-autocompletion") {
+		return
+	}
+
+	if runtime.GOOS == "windows" {
+		return
+	}
+
+	if completeinstall.IsInstalled("mc") {
+		return
+	}
+
+	for {
+		fmt.Printf("Install mc auto-completion in your shell ? (y/n): ")
```

Comment:
We should disable the prompt if `mc` is running as a script or --json, --quiet is specified.

Reply:
Perhaps no-autocompletion should be true in case of running as scripts...

Reply:
> We should disable the prompt if `mc` is running as a script or --json, --quiet is specified.

Oups yes, fixed !

> Perhaps no-autocompletion should be true in case of running as scripts...

 Not sure how to detect if this is a script or not

Reply:
A script mode is basically non terminal, --json or --quiet mode.

Reply:
> A script mode is basically non terminal, --json or --quiet mode.

Yes fixed then